build.assets/Dockerfile

```diff
@@ -50,13 +50,18 @@ RUN (mkdir -p /opt && cd /opt && curl https://storage.googleapis.com/golang/go1.
 RUN groupadd jenkins --gid=$GID -o && useradd jenkins --uid=$UID --gid=$GID --create-home --shell=/bin/sh
 RUN (mkdir -p /var/lib/teleport && chown -R jenkins /var/lib/teleport)
 
+# get etcd
+RUN (curl -L https://github.com/coreos/etcd/releases/download/v3.0.16/etcd-v3.0.16-linux-amd64.tar.gz | tar -xz ;\
```

Comment:
I am pushing back because I actually spent a ton of time simplifying our build-test chain of scripts for Teleport, the change  you're requesting adds some of that complexity back, but give me the benefit?